main.js

```diff
@@ -172,6 +172,12 @@ app.on('window-all-closed', function () {
   }
 })
 
+app.on('activate', function() {
+  if (!Object.keys(windows).length) {
+    init()
```

Comment:
Ah interesting. I'll fix that up really quickly. Out of curiosity, are there background windows being loaded?